src/lib/update-alternatives/model/alternative.rb

```diff
@@ -90,16 +90,25 @@ def self.load_choices_from(data)
 
     def self.load_choice(data)
       map = to_map(data)
-      Choice.new(map[:path], map[:priority], "")
+      Choice.new(map[:path], map[:priority], map[:slaves])
     end
 
     def self.to_map(choice_data)
       {
         path:     choice_data[0].split.last,
-        priority: choice_data[1].split.last
+        priority: choice_data[1].split.last,
+        slaves:   parse_slaves(choice_data[3..choice_data.length])
       }
```

Comment:
in general I think it would be more readable to process at first passed data and then create map. like:

```
path, priority, *slaves = choice_data
{
          path:     path.split.last,
          priority: priority.split.last
          slaves:   parse_slaves(slaves)
}
```